backend/controllers/LocationsController.ts

```diff
@@ -93,6 +94,31 @@ const LocationsController = {
       return res.status(500).json({ error: err.message });
     }
   },
+
+  async addLocationComments(req: Request, res: Response) {
+    try {
+      const { id,comment } = req.body;
+      const commentProperties = ['author','text','likes','dislikes','createdAt','updatedAt'];
+      let fullComment:boolean = true;
+      const location = await Location.findById(id);
+      if ( !location||!location.comments ) {
+        return res.status(400).json({ error: "Location or comments doesn't exist" });
+      }
+     location.comments.push(comment)
+     for ( let i in commentProperties ) {  if ( !comment.hasOwnProperty(commentProperties[i]) ){
+      fullComment = false;
+      return ( res.status(410).json({ error: `comment doesn't have ${commentProperties[i]} propertie` }),fullComment)
+     }}
```

Comment:
could be simplified

commentProperties.forEach(field => {
    if (!Object.keys(comment).includes(field))
        return res.status(410).json({ error: `comment doesn't have ${field} property` });
});